### ISSUE TITLE
Don't show "badge icon" in system tray & taskbar when in DnD mode

### ISSUE THREAD

**Issue by CombeeMike** [getferdi/ferdi#871]
**Is your feature request related to a problem? Please describe.**
Right now, when in DnD mode, Ferdi still updates the tray & taskbar icon when there are services with unread badges. When in DnD, I'd like to work without any distraction at all. This little red circle distracts me enough to make me open Ferdi to see whats going on.

**Describe the solution you'd like**
When in DnD, show some dedicated icon in the tray & taskbar makes it clear at one glance that I'm in DnD (blur icon, make it monochrome in taskbar, ...). This icon should always stay the same, no matter if there are unread badges or not.

You could make this optional to not change existing behavior for users who are already used to this.

**Additional context**
In short, I want to see at 1 glance when Ferdi is in DnD & I don't want to get any notification about new stuff whatsoever when in DnD.

**Comment by CombeeMike** [comment]
I'm still interested in this feature. Just for the sake of completeness 😉
